engine/runtime/src/main/scala/org/enso/interpreter/instrument/command/OpenFileCmd.scala

```diff
@@ -17,16 +17,16 @@ class OpenFileCmd(request: Api.OpenFileNotification) extends Command(None) {
     ec: ExecutionContext
   ): Future[Unit] =
     Future {
-      ctx.locking.acquireFileLock(request.path)
       ctx.locking.acquireReadCompilationLock()
```

Comment:
What is a meanng of `ReadCompilationLock`? No compilation can happen while I hold the "read access"?

Reply:
It appears so, yes.